input/fsh/VS_Radiotherapy.fsh

```diff
@@ -17,7 +17,8 @@ Title: "Brachytherapy Modality Value Set"
 Description: "Codes describing the modalities of brachytherapy procedures."
 * ^copyright = "This value set includes content from SNOMED CT, which is copyright © 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement"
 * SCT#1156708005 "Low dose rate brachytherapy using temporary radioactive source (procedure)"
-* SCT#169359004 "Internal radiotherapy - permanent seeds (procedure)"
+* SCT#1156707000 "Low dose rate brachytherapy (procedure)"
```

Comment:
I don't recall the discussion where we said we need LDR (1156707000), in addition to LDR-Temp (1156708005) and LDR-Perm (169359004). I think it is sufficient to have LDR-Temp and LDR-Perm, exactly as in the ballot version. I believe we requested LDR only because it provides the parent for LDR-Perm and LDR-Temp.

Reply:
Codex RTTD decided -- LDR-Temp and LDR-Perm terms are sufficient.  LDR term will be deleted.